rails-response-dumper.gemspec

```diff
@@ -2,7 +2,7 @@
 
 Gem::Specification.new do |spec|
   spec.name = 'rails-response-dumper'
-  spec.version = '5.3.0'
+  spec.version = '5.4.0'
```

Comment:
We normally handle incrementing the version when we're ready to release, not in a feature change.

Reply:
Reverted in [Revert version bump](https://github.com/Pioneer-Valley-Books/rails-response-dumper/pull/68/commits/16ef06da4925226a23dc0b9626c4d013a4b35ed9)